attributes/default.rb

```diff
@@ -87,3 +87,20 @@
     group:    ['admin'],
   },
 }]
+
+default['rundeck_server']['rundeck-config.framework']['framework.server.name '] = 'localhost'
```

Comment:
there's an extra space here and the following attributes for `framework.server.name `